src/components/home/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+
+import Button from "../base/button";
+import CreateTask from "../task/create-new";
+import TaskCard from "../task/existing-card";
+import Styles from "./index.module.scss";
+
+function HomeContainer() {
+  const tasks = useSelector((state) => state.todo.tasks);
+  const [isNewTaskRequested, setIsNewTaskRequested] = useState(false);
```

Comment:
We can move states into our redux-store from the beginning if we think that the state might get shared by different Components as our App grows, in the future.

Reply:
**isNewTaskRequested** is a flag used to display only the **new-task** card inside this component & till now no other component display this card. So I feel that transferring this var to redux will increase complexity & dependency. @fza-wtag vai